### DISCUSSION TITLE
Enable topic deletion

### DISCUSSION THREAD

**Discussion by Fran-Rg** [provectus/kafka-ui#3530] [Q&A] [answered]
I'm using AWS Kafka serverless (don't know if that matters), I've set the cluster with: `KAFKA_CLUSTERS_0_READONLY : false`

But I'm still unable to delete the topic, I get:
The topic deletion is restricted at the application configuration level

I found the code for the UI that does that but I couldn't get to the bottom to how to change it: https://github.com/provectus/kafka-ui/blob/738136eed281b14c36194e18236c550f73a4fdaa/kafka-ui-react-app/src/components/Topics/List/ActionsCell.tsx#L85-L92

**Comment by Haarolean** [comment, marked as answer]
Hi, there might be an issue indeed. Please follow #3536 for the updates.
